4.1/crud-fields.md

```diff
@@ -908,6 +908,10 @@ Class Product extends Model
             $public_destination_path = Str::replaceFirst('public/', '', $destination_path);
             $this->attributes[$attribute_name] = $public_destination_path.'/'.$filename;
         }
+	else
+        {
+            return $this->attributes['image'] = $value;
+        }
```

Comment:
```suggestion
        else
        {
            return $this->attributes[$attribute_name] = $value;
        }
```

Reply:
Hi @MerdanDev, thank you for this suggestion, I approve it 👌
I have just this small changes to apply, and I see you've deleted the repo from your side, so I cannot apply the changes 😅, can you please try to somehow recover it?